src/Services/Analytics.ts

```diff
@@ -0,0 +1,96 @@
+import { GA } from "../Utils/GA";
+
+type Options = {
+  ga: {
+    trackingId: string;
+    appName: string;
+    env: string;
+  };
+};
+
+type TransferEventData = {
+  address: string;
+  recipient: string;
+  amount: number;
+  nonce?: number;
+};
+
+class AnalyticsService {
+  ga: GA;
+
+  constructor(options: Options) {
+    this.ga = new GA(options.ga);
+  }
+
+  transferIntransitEvent({
+    address,
+    recipient,
+    nonce,
+    amount,
+  }: TransferEventData) {
+    this.ga.event("transfer_intransit", {
+      address,
+      recipient,
+      nonce,
+      amount,
+    });
+  }
+
+  transferCompletedEvent({
+    address,
+    recipient,
+    nonce,
+    amount,
+  }: TransferEventData) {
+    this.ga.event("transfer_completed", {
+      address,
+      recipient,
+      nonce,
+      amount,
+    });
+  }
+
+  fallbackTransferCompletedEvent({
```

Comment:
`trackTransferCompletedFromFallback`?
Please, apply to all fallback methods.

Reply:
fixed